Chewy swap patch 1

ChewySwap opened the pull request:
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama): ChewySwap


##### Twitter Link: https://twitter.com/ChewySwap


##### List of audit links if any:  https://github.com/ContractWolf/smart-contract-audits/blob/main/ContractWolf_Audit_ChewySwap_DEX.pdf
https://github.com/ContractWolf/smart-contract-audits/blob/main/ContractWolf_Audit_DogeShrekDEX.pdf


##### Website Link: https://chewyswap.dog


##### Logo (High resolution, will be shown with rounded borders): https://chewyswap.dog/images/512x512.png


##### Current TVL: ~$275,000


##### Treasury Addresses (if the protocol has treasury)


##### Chain: Shibarium and Dogechain


##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): chewyswap


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)


##### Short Description (to be shown on DefiLlama): ChewySwap is the leading Community-Based decentralized exchange on both Shibarium and Dogechain, the two leading EVM compatible blockchains for Dog lovers!


##### Token address and ticker if any: 0x2761723006d3Eb0d90B19B75654DbE543dcd974f CHEWY


##### Category (full list at https://defillama.com/categories) *Please choose only one: Dexes


##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):


##### forkedFrom (Does your project originate from another project): UniswapV2


##### methodology (what is being counted as tvl, how is tvl being calculated): Factory contract liquidity balance


##### Github org/user (Optional, if your code is open source, we can track activity): https://github.com/chewyswap

g1nt0ki commented on projects/ChewySwap/index.js:
```diff
@@ -0,0 +1,10 @@
+    tvl: getUniTVL({ factory: "0x5640113EA7F369E6DAFbe54cBb1406E5BF153E90", useDefaultCoreAssets: true,})
+  },
+  dogechain: {
+    tvl: getUniTVL({ factory: "0x7C10a3b7EcD42dd7D79C0b9d58dDB812f92B574A", useDefaultCoreAssets: true,})
```
this factory belongs to DogeShrek? https://defillama.com/protocol/doge-shrek

g1nt0ki commented on projects/ChewySwap/index.js:
```diff
@@ -0,0 +1,10 @@
+
+module.exports = {
+  shibarium: {
+    tvl: getUniTVL({ factory: "0x5640113EA7F369E6DAFbe54cBb1406E5BF153E90", useDefaultCoreAssets: true,})
```
this belongs to punkswap? https://defillama.com/protocol/punkswap

ChewySwap commented:
Oof, I'm so sorry it was late when I did this PR. I meant to put 
chewyswap's factory in there and Dogeshrek has actually being merged 
into ChewySwap. I took over operations of the dex back in July and we 
rebranded. So the Dogechain factory is correct the Dogeshrek page can be 
removed and replaced with ChewySwap.

ChewySwap Shibarium: 0xEDedDbde5ffA62545eDF97054edC11013ED72125

ChewySwap Dogechain: 0x7C10a3b7EcD42dd7D79C0b9d58dDB812f92B574A
(Formerly Dogeshrek)

https://docs.chewyswap.com/developers/smart-contracts/chewyswap-exchange

So sorry for the mixup didn't mean to create more work for you. Should I 
update the PR? As proof of the merge if you look at the read contract 
page of each factory contract you will see they have the same feeTo 
address which is the deployer wallet for ChewySwap and new dev wallet 
for Dogeshrek. If there's anything else I need to do to show proof of 
the merge let me know.

Thanks,
Jim


On 2023-10-24 07:28, g1nt0ki wrote:
> @g1nt0ki requested changes on this pull request.
> -------------------------
> 
> In projects/ChewySwap/index.js [1]:
> 
>> @@ -0,0 +1,10 @@
> +const { uniTvlExport } = require('../helper/unknownTokens')
> +
> +module.exports = {
> +  shibarium: {
> +    tvl: getUniTVL({ factory:
> "0x5640113EA7F369E6DAFbe54cBb1406E5BF153E90", useDefaultCoreAssets:
> true,})
> +  },
> +  dogechain: {
> +    tvl: getUniTVL({ factory:
> "0x7C10a3b7EcD42dd7D79C0b9d58dDB812f92B574A", useDefaultCoreAssets:
> true,})
> 
> this factory belongs to DogeShrek?
> https://defillama.com/protocol/doge-shrek
> -------------------------
> 
> In projects/ChewySwap/index.js [2]:
> 
>> @@ -0,0 +1,10 @@
> +const { uniTvlExport } = require('../helper/unknownTokens')
> +
> +module.exports = {
> +  shibarium: {
> +    tvl: getUniTVL({ factory:
> "0x5640113EA7F369E6DAFbe54cBb1406E5BF153E90", useDefaultCoreAssets:
> true,})
> 
> this belongs to punkswap? https://defillama.com/protocol/punkswap
> 
> --
> Reply to this email directly, view it on GitHub [3], or unsubscribe
> [4].
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> 
> Links:
> ------
> [1] 
> https://github.com/DefiLlama/DefiLlama-Adapters/pull/7822#discussion_r1369643112
> [2] 
> https://github.com/DefiLlama/DefiLlama-Adapters/pull/7822#discussion_r1369643803
> [3] 
> https://github.com/DefiLlama/DefiLlama-Adapters/pull/7822#pullrequestreview-1694053027
> [4] 
> https://github.com/notifications/unsubscribe-auth/BCBD4ERSU7GLGSL2VZNWKRTYA5GYRAVCNFSM6AAAAAA6M7CNR2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOJUGA2TGMBSG4

g1nt0ki commented:
closing as https://github.com/DefiLlama/DefiLlama-Adapters/pull/7829
